### DISCUSSION TITLE
osu! repeatedly closing (Ubuntu Impish Indri)

### DISCUSSION THREAD

**Discussion by X4X5** [ppy/osu#14995] [Q&A]
I'm not exactly sure how to explain this, but I ran osu!lazer, and for some reason, it keeps minimizing the game, but the game still runs. Not sure what log this is under, so I zipped up all the log files
[logs.zip](https://github.com/ppy/osu/files/7307674/logs.zip)

Current OS: Ubuntu Impish Indri (21.10, released yesterday)
osu!lazer version: 2021.1006.1

**Comment by gagahpangeran** [comment]
Can you please provide video about your issue?

**Comment by X4X5** [comment]
https://drive.google.com/file/d/1iKKYfLzvgSThcCkj13bAks35m48h8GmB/view?usp=sharing

**Reply by gagahpangeran** [reply]
Is that on wayland or xorg session?

**Reply by X4X5** [reply]
wayland

**Reply by bdach** [reply]
wayland is not officially supported and may be broken in various ways. patches/PRs to fix are welcome but this is not likely be an immediate priority for the team.

(some of it may be SDL-level breakage even.)
